integration/shareddata/scalars.go

```diff
@@ -28,7 +28,10 @@ const (
 	doubleBig     = float64(1 << 61)   // 2305843009213693952.0: some number larger than safe integer (doubleBig+1 == doubleBig)
 	longBig       = int64(1 << 61)     // 2305843009213693952:   same as doubleBig but integer
 
-	// TODO https://github.com/FerretDB/FerretDB/issues/2321
+	doubleMaxOverflow = float64(9223372036854776833)  // double value that is larger than MaxInt64
```

Comment:
Could you also check these int64? I think I moved them to shareddata when we move from integration to compat tests. But I don't know the significance of those numbers :thinking: 
```
		"int64-1":          int64(1099511628000),
		"int64-2":          int64(281474976700000),
		"int64-3":          int64(72057594040000000),
```
,
```
		"int32-1": int32(4080),
		"int32-2": int32(1048560),
		"int32-3": int32(268435440),
```

and
```
	"double-1":     4080.1234,
		"double-2":     1048560.0099,
		"double-3":     268435440.2,
```

Reply:
Looking at PR title, maybe there will be another PR for handling non-double numbers?

Reply:
I\ve checked all of those values and it seems to me that none of them represent any specific number.

Reply:
So let's add a comment about that. `// random` or something like that